crates/cli/src/lib.rs

```diff
@@ -469,17 +481,38 @@ fn find_ext(manifest: Option<&PathBuf>) -> AResult<cargo_metadata::Target> {
 ///
 /// * `target` - The target to compile.
 /// * `release` - Whether to compile the target in release mode.
+/// * `features` - Optional list of features.
 ///
 /// # Returns
 ///
 /// The path to the target artifact.
-fn build_ext(target: &Target, release: bool) -> AResult<Utf8PathBuf> {
+fn build_ext(
+    target: &Target,
+    release: bool,
+    features: Option<Vec<String>>,
+    all_features: bool,
+    no_default_features: bool,
+) -> AResult<Utf8PathBuf> {
     let mut cmd = Command::new("cargo");
     cmd.arg("build")
         .arg("--message-format=json-render-diagnostics");
     if release {
         cmd.arg("--release");
     }
+    if let Some(features) = features {
+        cmd.arg("--features");
+        for feature in features {
+            cmd.arg(feature);
+        }
+    }
+
+    if all_features {
+       cmd.arg("--all-features");
+    }
+
+    if no_default_features {
+        cmd.arg("--no-default-features");
+    }
```

Comment:
I don't if it is an edge case, but this kind like of exclusives flags when these two cannot co-exist at the same time

Reply:
Well I am letting cargo handle that. Just passing those as-is.